opengrok-web/src/test/java/org/opengrok/web/api/v1/controller/SystemControllerTest.java

```diff
@@ -130,4 +136,26 @@ public void testDtagsEftarReload() throws IOException {
         // Cleanup
         IOUtils.removeRecursive(dataRoot);
     }
+
+    @Test
+    public void testIndexTime() throws IOException, ParseException {
+        Path dataRoot = Files.createTempDirectory("indexTimetest");
+        env.setDataRoot(dataRoot.toString());
+        File indexTimeFile = Paths.get(dataRoot.toString(), "timestamp").toFile();
```

Comment:
I recommend using:
`dataRoot.resolve("timestamp")`
and then methods from
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Files.html

Reply:
The `resolve()` is beneficial. Not so sure about the `Files` methods.